packages/esm-appointments-app/src/appointments/common-components/appointments-actions.component.tsx

```diff
@@ -18,7 +18,6 @@ dayjs.extend(isToday);
 
 interface AppointmentActionsProps {
   appointment: Appointment;
-  scheduleType: string;
```

Comment:
Not sure this scheduleType property was being properly used or if it was even needed in the long run.  I think the main usage was to render a "Add appointments" button in the "Today's Appointments" view.  We could bring that back if necessary.